riff-raff.yaml

```diff
@@ -15,7 +15,7 @@ deployments:
     parameters:
       amiTags:
         BuiltBy: amigo
-        Recipe: editorial-tools-focal-java8-ARM-WITH-cdk-base
+        Recipe: editorial-tools-focal-java17
```

Comment:
Choosing Java 17 here as it seems to be the most recent [AMIgo image](https://amigo.gutools.co.uk/recipes/editorial-tools-focal-java17) available - would have chosen a Java 21 LTS AMIgo image if it was available!